### PR TITLE
Changing 'latency' channel to be UoM Number:Time

### PR DESCRIPTION
**Pull request by clinique**
Signed-off-by: Gaël L'hopital <gael@lhopital.org>

### CODE REVIEW

**Comment by clinique** [comment]
This can only be merged once [this PR](https://github.com/openhab/openhab-core/pull/798) is accepted

File: bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java [hunk 1]
```diff
@@ -116,7 +119,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void partialDetectionResult(PresenceDetectionValue value) {
         updateState(CHANNEL_ONLINE, OnOffType.ON);
-        updateState(CHANNEL_LATENCY, new DecimalType(value.getLowestLatency()));
+        updateState(CHANNEL_LATENCY, new QuantityType<>(value.getLowestLatency(), SmartHomeUnits.MILLISECOND));
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java]: Same question like in https://github.com/openhab/openhab-core/pull/798#issuecomment-489577501: 
I am not sure if we really need it. Doesn't `MILLI(SECOND)` do the job at all?

**Comment by J-N-K** [comment]
Is this considered a breaking change? Users need to update their items.

File: bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java]: Yes, I cancelled PR in OH core and will do this.

**Comment by clinique** [comment]
> Is this considered a breaking change? Users need to update their items.

Yes, this is a breaking change.

**Pull request merged**